Sources/LogtoClient/LogtoClient/LogtoClient.swift

```diff
@@ -10,14 +10,18 @@ import Logto
 
 public class LogtoClient {
     internal let authContext = LogtoAuthContext()
-    internal let accessTokenMap: [String: AccessToken] = [:]
     internal let logtoConfig: LogtoConfig
     internal let networkSession: NetworkSession
 
+    internal var accessTokenMap: [String: AccessToken] = [:]
     internal var idToken: String?
     internal var refreshToken: String?
     internal var oidcConfig: LogtoCore.OidcConfigResponse?
 
+    internal func buildAccessTokenKey(for resource: String = "", scopes: [String]) -> String {
+        "\(scopes.sorted().joined(separator: " "))@\(resource)"
```

Comment:
Sorting is unnecessary, right?

Reply:
in this case sorting matters, it's for the cache key

Reply:
ok